libs/components/tabs/src/lib/modules/tabs/tabset.component.spec.ts

```diff
@@ -627,6 +645,7 @@ describe('Tabset component', () => {
 
     function fireResizeEvent() {
       SkyAppTestUtility.fireDomEvent(window, 'resize');
```

Comment:
Is `SkyAppTestUtility.fireDomEvent(window, 'resize');` even needed for this test anymore?